api/registrations/views.py

```diff
@@ -850,3 +856,32 @@ def get_default_queryset(self):
 
     def get_queryset(self):
         return self.get_queryset_from_request()
+
+
+class RegistrationSchemaResponsesList(JSONAPIBaseView, ListFilterMixin, generics.ListCreateAPIView, RegistrationMixin):
+    required_read_scopes = [CoreScopes.NULL]
+    required_write_scopes = [CoreScopes.NULL]
+
+    permission_classes = (
+        drf_permissions.IsAuthenticated,
+        base_permissions.TokenHasScope,
+        AdminContributorOrPublic,
+    )
+
+    view_category = 'registrations'
+    view_name = 'schema-responses-list'
+
+    serializer_class = SchemaResponsesListSerializer
+
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=json.loads(self.request._request._body))
```

Comment:
I could not find a way to pass this data without overriding this method and shoving the data in there.

Reply:
We probably don't need this function. I'm okay with saying we should only be POSTing to the top-level endpoint for SchemaResponses.